Content.Shared/Item/UserInteractedWithItemEvent.cs

```diff
@@ -0,0 +1,17 @@
+namespace Content.Shared.Item;
+
+/// <summary>
+/// Raised on the user after they do any sort of interaction with an item,
+/// useful for when you want a component on the user to do something to the item.
+/// E.g. forensics, disease, etc.
+/// </summary>
+public sealed class UserInteractedWithItemEvent : EntityEventArgs
+{
+    public EntityUid User;
+    public EntityUid Item;
```

Comment:
Might as well make it a ref event given it's all readonly value types anyway.

Reply:
I don't know what that means.

Reply:
don't really think this is necessary